package/CHANGELOG

```diff
@@ -13,12 +13,13 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/?? IAlibay, PicoCentauri, orbeckst
+??/??/?? IAlibay, PicoCentauri, orbeckst, Tengyu Xie
```

Comment:
```suggestion
??/??/?? IAlibay, PicoCentauri, orbeckst, miss77jun
```

Typically we ask for the GitHub username here, just so we can track contributions being added :)